G/GAP_lib/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase, Pkg
```

Comment:
This isn't a big deal, but I don't think BinaryBuilderBase is needed here

Reply:
Another copy&paste from GAP.jl

Reply:
(actually I am not sure why I need it there either? A leftover from a previous iteration of the code?)